spec/cc/engine/analyzers/engine_config_spec.rb

```diff
@@ -84,6 +85,18 @@ module CC::Engine::Analyzers
 
         assert_equal engine_config.mass_threshold_for("elixir"), 13
       end
+
+      it "returns nil when language is empty" do
+        engine_config = EngineConfig.new({
+          "config" => {
+            "languages" => {
+              "ruby" => "",
+            }
+          }
+        })
+
+        assert_equal engine_config.mass_threshold_for("ruby"), nil
```

Comment:
I'm confused by this spec. Above you change `paths_for` but this spec tests `mass_threshold_for`. Should there be another spec?